src/IO/ZarrMultiscaleSpatialImage.js

```diff
@@ -204,6 +206,7 @@ class ZarrMultiscaleSpatialImage extends MultiscaleSpatialImage {
     const chunkPathBase = info.pixelArrayPath
     const chunkPaths = []
     const chunkPromises = []
+    const chunksQueue = new PQueue({ concurrency: 5 })
```

Comment:
Can we bump this up to something higher, e.g. `10`, `16`, `32`?

Reply:
Oof, yes, that was a mistake. I actually had it at 10 but some more testing looks like we can set it to 16 with consistent results. Values above that are inconsistent. Fixed to be 16 now.